### PR TITLE
First page setup and Cards component

### PR DESCRIPTION
**Pull request by ncbecker**
- Add GlobalStyle and background image
- Create Header and Footer
- Create "frosted glass" Cards component with Intersection Observer API to animate slide-in on scroll

### CODE REVIEW

File: src/Footer.tsx [hunk 1]
```diff
@@ -0,0 +1,32 @@
...
+  display: flex;
+  justify-content: flex-end;
+  backdrop-filter: blur(2.5rem);
+`;
+
+export const Footer = (): JSX.Element => {
+  return (
+    <Blur>
+      <span>
+        Background photo by{" "}
```
Comment (lauravikanis) [inline src/Footer.tsx:18]:
```suggestion
        Background photo by
```
You can remove all the {" "}

File: src/index.tsx [hunk 2]
```diff
@@ -1,17 +1,10 @@
...
     <App />
   </React.StrictMode>,
-  document.getElementById('root')
+  document.getElementById("root")
 );
-
-// If you want to start measuring performance in your app, pass a function
-// to log results (for example: reportWebVitals(console.log))
-// or send to an analytics endpoint. Learn more: https://bit.ly/CRA-vitals
-reportWebVitals();
```
Comment (lauravikanis) [inline src/index.tsx:17]: Why remove this part? [Measuring Performance](https://create-react-app.dev/docs/measuring-performance/)

**Review by lauravikanis — APPROVED** [review]
LGTM! Looking forward to seeing the finished project :)

File: src/Footer.tsx (hunk 1 above)
Comment (ncbecker) [reply src/Footer.tsx:18]: I need these for spaces between the span and anchor tags 👀

File: src/index.tsx (hunk 2 above)
Comment (ncbecker) [reply src/index.tsx:17]: Good question :) I thought I don't need these, but will look into it

**Pull request merged**